pkg/eval/sys/sys_test.go

```diff
@@ -0,0 +1,17 @@
+package sys
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/elves/elvish/pkg/eval"
+)
+
+func TestMath(t *testing.T) {
+	setup := func(ev *eval.Evaler) { ev.Builtin.AddNs("sys", Ns) }
+	That := eval.That
```

Comment:
This is really a nitpick, but I'd declare `That` as a global variable to be consistent with other test files:

Reply:
Fair enough. I copied that pattern from *./pkg/eval/re/re_test.go*. I'll create another PR to fix that inconsistency. :smile:

Reply:
FWIW, It appears that *./pkg/eval/re/re_test.go* is the only other unit test which employs this pattern. All of the other unit test files which use `That()` are in the `eval` package and thus do not need an alias.

Reply:
The `That` function reference can't be made a global in the unit tests outside the `eval` package, AFAICT.